_rest_routes.inc.php

```diff
@@ -393,6 +394,10 @@
         RestConfig::authorization_check("patients", "med");
         return (new FhirAllergyIntoleranceRestController(null))->getOne($id);
     },
+    "GET /fhir/Observation/:id" => function ($id) {
+        RestConfig::authorization_check("patients", "med");
+        return (new FhirObservationRestController(null))->getOne($id);
+    },
```

Comment:
I can't think of a specific id for an Observation. All observation endpoints should come from a search by:

- patient + category
- patient + category + date
- patient + category + code
- patient + category + code + date
- patient + _profile

also could be patient + encounter
encounter + category and so forth


You get the idea and these we should support but some searches shall be supported.
Look at spec for those.

Reply:
> * Up Next -> Patient wise Search
> * Up Next -> Category wise Search

Few are already planned and will try to add more if possible 👍

Reply:
> I can't think of a specific id for an Observation. All observation endpoints should come from a search by:

Test Server and [Examples ](http://hl7.org/fhir/R4/observation-example-f202-temperature.json.html) Gives id in the resource and I think ID will be necessary for anybody to locate a specific observation.

Reply:
Although http://hl7.org/fhir/R4/bodyweight.html it doesn't show as required but as we have already done it which mean we are capable of this. Let me know your thought 🤔 ?

Reply:
> Gives id in the resource

That's because those are examples of a saved Observation resource which could then be ref by the saved id.
We don't save specific observations to give an id. This is because we are not a Fhir repository/server. All our Observation will be built from various core areas of openemr. 

I know this can get confusing as we are kind of walking a thin line with what we can and can not support. Try to think about our use case as being a client to a Fhir server.


> Although http://hl7.org/fhir/R4/bodyweight.html it doesn't show as required

I was talking about what searches are required for a given resource. Not content...